cedar-policy-validator/src/err.rs

```diff
@@ -192,3 +192,93 @@ pub enum UnsupportedFeature {
     #[error("action declared with attributes: [{}]", .0.iter().join(", "))]
     ActionAttributes(Vec<String>),
 }
+
+// PANIC SAFETY testing code
+#[allow(clippy::panic)]
+#[cfg(test)]
+mod test {
```

Comment:
nit: most schema parsing tests live in `schema.rs`

Reply:
Ah sure thing, I'll move them